docs/Guides/Reviews.md

```diff
@@ -0,0 +1,59 @@
+---
+tags: [reviews]
+---
+
+# Reviews
+
+A review of an item - for example, of a restaurant, movie, or store.
+
+## Common Actions
+
+### View reviews for a business location
```

Comment:
Do we want to consider using entity so we are able to expand these APIs for service area businesses and professionals?